modules/openapi-generator/src/main/resources/cpp-pistache-server/cmake.mustache

```diff
@@ -20,7 +20,6 @@ ExternalProject_Add(NLOHMANN
 
 )
 include_directories(${EXTERNAL_INSTALL_LOCATION}/include)
-include_directories(${EXTERNAL_INSTALL_LOCATION}/include/nlohmann)
```

Comment:
why remove this? it removes a manual step

Reply:
ahhh maybe you are including as `nlohman/json.hpp`

Reply:
Yeah I include nlohmann/json.hpp because that's what they recommend in their documentation and it's how all the package managers I've used package it.